app/services/LocationService.js

```diff
@@ -246,8 +246,17 @@ export default class LocationServices {
       } else {
         BackgroundGeolocation.start(); //triggers start on start event
 
-        // TODO: We reach this point on Android when location services are toggled off/on.
-        //       When this fires, check if they are off and show a Notification in the tray
+        BackgroundGeolocation.checkStatus(({ locationServicesEnabled }) => {
+          if (!locationServicesEnabled) {
+            PushNotification.localNotification({
+              id: '55',
```

Comment:
What is this number?

Reply:
This is random number to identify notification for ability cancel it from code.

Reply:
OK, please make a meaningful `const NOTIFICATION_X = 'blah'` for it